test/Broadway/CommandHandling/ClosureCommandHandlerTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+
+namespace Broadway\CommandHandling;
+
+use Broadway\TestCase;
+
+class ClosureCommandHandlerTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function it_delegates_command_to_proper_handle_function()
+    {
+        $commandHandler = new TestClosureCommandHandler();
+        $command        = new ClosureCommandHandlerTestCommand();
+        $commandHandler->handle($command);
+
+        $this->assertTrue($commandHandler->handled);
+    }
+
+    /**
+     * @test
+     *
+     * @dataProvider unresolvableCommands
+     */
+    public function handle_should_throw_exception_when_impossible_to_delegate_to_a_valid_closure($command)
```

Comment:
what you're testing here is that the handle method only accepts object, right?

suggestion: it_throws_when_handling_a_non_object_command()